Sources/IO/Geometry/STLReader/index.js

```diff
@@ -103,7 +103,9 @@ function vtkSTLReader(publicAPI, model) {
 
   // Internal method to fetch Array
   function fetchData(url, option = {}) {
-    const { compression, progressCallback } = model;
+    const compression = option.compression || model.compression;
+    const progressCallback = option.progressCallback || model.progressCallback;
```

Comment:
I would make the difference between `option.progressCallback === undefined` and `option.progressCallback === null`.
In the latter case, it means that we do not want progress callback, in the former case, it means we want the model progressCallback.
Hence:
```
const compression = option.compression !== undefined ? option.compression : model.compression;
const progressCallback = option.progressCallback !== undefined ? option.progressCallback : model.progressCallback;
```